internal/configs/named_values.go

```diff
@@ -123,6 +125,19 @@ func decodeVariableBlock(block *hcl.Block, override bool) (*Variable, hcl.Diagno
 		v.SensitiveSet = true
 	}
 
+	if attr, exists := content.Attributes["deprecated"]; exists {
+		valDiags := gohcl.DecodeExpression(attr.Expr, nil, &v.Deprecated)
+		if strings.TrimSpace(v.Deprecated) == "" {
```

Comment:
Missing a check for valDiags.HasErrors()

Reply:
Good catch. Thanks